### ISSUE TITLE
Plugins type definition in bower.json needs to be more self-explanatory

### ISSUE THREAD

**Issue by taylortom** [adaptlearning/adapt_framework#964]
https://github.com/adaptlearning/adapt_authoring/issues/1043

**Comment by oliverfoster** [comment]
+1

**Comment by oliverfoster** [comment]
might have to do both for a time beng?

**Comment by taylortom** [comment]
Yep, happy to do that

**Comment by moloko** [comment]
could we also add fields to state whether the plugin:
* is authoring tool compatible
* has accessibility support
* has RTL support
?

**Comment by moloko** [comment]
Actually I suppose the last two could be included in the README - but I think the AT compatibility one would be useful. Also, you might want to search only for plugins that are AT-compatible, that would make this possible.

**Comment by taylortom** [comment]
I think all of those are vital for any plugin browsers (including the one of the community site).

**Comment by moloko** [comment]
this has been sitting around for a year now, anything further on this?

**Comment by oliverfoster** [comment]
i'm a crossover adapt-cli / AT / framework thing that we should defo still do. Just needs some political will.

**Comment by taylortom** [comment]
Yep, still keen to get this in, but will prompt some reworking of the AT code, so needs to be discussed.

**Comment by oliverfoster** [comment]
I reckon we could do the adapt-cli bits first. Then the plugins, then the AT. Both styles should run concurrently to avoid any funny issues.
